CRAMv3.tex

```diff
@@ -1708,36 +1894,42 @@ \subsubsection*{Introduction}
 
 \subsubsection*{Offset}
 
-Most of the codings listed below encode positive integer numbers. An integer offset 
+Many of the codings listed below encode positive integer numbers. An integer offset 
 value is used to allow any integer numbers and not just positive ones to be encoded. 
 It can also be used for monotonically decreasing distributions with the maximum 
 not equal to zero. For example, given offset is 10 and the value to be encoded 
 is 1, the actually encoded value would be offset+value=11. Then when decoding, 
 the offset would be subtracted from the decoded value. 
 
-\subsubsection*{Beta coding}
+\subsubsection{External: codec ID 1}
 
-\subsubsection*{Definition}
+Can encode types \textit{Byte}, \textit{Integer}.
```

Comment:
+ Long

Reply:
I've added Long to io_lib for my CRAMv4 prototype and I see it's in htsjdk too, but htslib has no support for external long.

This section is ambiguous though as it describes Int and Byte being acceptable and then mentions Long in the next sentence.  In practice CRAM v3 (and before) have no longs in any data series, with long restricted purely to ltf8 encoding within container and slice headers.

We should be consistent though.  For v3 I think just drop long from external as it's not a possibility.

Reply:
OK.  The HTSJDK implementation of External Long is broken anyway.